zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+The Financial Privacy Foundation (FPF) would administer the Coin-holder-controlled Fund based on decisions taken by coin-holder voting, following a model decided by the community and specified in another ZIP [TBD].
+
+### Disbursement process
+
+1. Anyone can submit a grant application via a process agreed between the FPF, the Zcash Foundation, and the Electric Coin Company.
+2. Grant applications with total below USD 500,000 (or equivalent in another currency) would be directed to ZCG.
+3. For grant applications above this threshold, a 30-day community review and feedback period would start.
+4. If the grant applications should proceed to a coin-holder vote according to the agreed process, then coin-holders would be asked to vote on it.
+5. If the vote passes, then as payments are scheduled for the grant, then the FPF, the Zcash Foundation, and the Electric Coin Company SHOULD sign the corresponding transactions for disbursement from the Coin-holder-controlled Fund.
+
+For coin holder votes, a minimum of 420,000 ZEC (2% of the total eventual supply) MUST be voted, with a simple majority cast in favor, for a grant proposal to be approved. Upon approval, the grants would be paid to the recipient per the terms of the grant proposal.
+
+Coin holders would not be obligated to fund grants and MAY leave the funds at rest.
+
+No organizations or individuals would be restricted from voting their coins.
+
+## Administrative obligations and constraints
+
+All provisions of ZIP 1015 imposing obligations and constraints on Bootstrap Project, Electric Coin Company, Zcash Foundation, Financial Privacy Foundation, the ZCG Committee, and grant recipients relating to the previous ``FS_FPF_ZCG`` funding stream, SHALL continue in effect for *both* Zcash Community Grants and the Coin-holder-controlled Fund.
+
+These obligations and constraints will be extended to Shielded Labs and the other Key-holder Organizations in respect of the Coin-holder-controlled Fund.
+
+The provisions after the first paragraph of the section "Zcash Community Grants (ZCG)" also apply to FPF's administration of the Coin-holder-controlled Fund, with coin-holder voting replacing the role of the ZCG Committee.
+
+For the avoidance of doubt, this includes the following:
+
+* Decisions on use of the Coin-holder-controlled Fund taken by coin-holder vote require no approval from the FPF Board, but are subject to veto if FPF judges them to violate Cayman law or the FPF's reporting requirements and other (current or future) obligations under the Cayman Islands' Companies Act (2023 Revision) and Foundation Companies Law, 2017.
+* FPF SHALL be contractually required to recognize the Coin-holder-controlled Fund as a Restricted Fund donation under the above constraints, and keep separate accounting of its balance and usage under its Transparency and Accountability obligations defined in ZIP 1015 [^zip-1015-transparency-and-accountability].
```

Comment:
If the FPF is not involved (as mentioned by Josh above) do these clauses need to be retained, or should they be modified to refer to the Key-Holder Organizations?

Reply:
They don't need to be retained given the section on Veto process.